docs/administrator-documentation/moderne-dx/how-to-guides/dx-configuration.md

```diff
@@ -312,6 +312,71 @@ export MODERNE_DX_ARTIFACTORY_0_PASSWORD=...
 export MODERNE_DX_MAVEN_0_USERNAME=...
 export MODERNE_DX_MAVEN_0_PASSWORD=...
 
+docker run \
+-e MODERNE_DX_TOKEN_0 \
+-e MODERNE_DX_ARTIFACTORY_0_URL=https://myartifactory.example.com/artifactory/ \
+-e MODERNE_DX_ARTIFACTORY_0_USERNAME \
+-e MODERNE_DX_ARTIFACTORY_0_PASSWORD \
+-e MODERNE_DX_ARTIFACTORY_0_ASTQUERYFILTERS_0='"name":{"$match":"*-ast.jar"}' \
+-e MODERNE_DX_ARTIFACTORY_0_ASTQUERYFILTERS_1='"repo":{"$eq":"example-maven"}' \
+-e MODERNE_DX_MAVEN_0_URL=https://myartifactory.example.com/artifactory/libs-releases-local \
+-e MODERNE_DX_MAVEN_0_LOCALREPOSITORY=~/.moderne-maven \
+-e MODERNE_DX_MAVEN_0_USERNAME \
+-e MODERNE_DX_MAVEN_0_PASSWORD \
+-e MODERNE_DX_ORGANIZATION_REPOSCSV=/Users/MY_USER/Documents/repos.csv \
+-e MODERNE_DX_ORGANIZATION_DEFAULTCOMMITOPTIONS=PullRequest,Branch,Direct \
+# ... Additional variables to come
+-p 8080:8080
+moderne-dx:latest
+```
+</TabItem>
+
+<TabItem value="executable-jar" label="Executable JAR">
+
+```bash
+# Exporting environment variables with the exact same structure as the parameter in the Java command makes it so you no longer need to include them in the below Java command. For instance, the first export below is equivalent to including this parameter in the Java command:
+# --moderne.dx.token[0]=...
+export MODERNE_DX_TOKEN_0=...
+export MODERNE_DX_ARTIFACTORY_0_USERNAME=...
+export MODERNE_DX_ARTIFACTORY_0_PASSWORD=...
+export MODERNE_DX_MAVEN_0_USERNAME=...
+export MODERNE_DX_MAVEN_0_PASSWORD=...
+
+java -jar moderne-dx-{version}.jar \
+--moderne.dx.artifactory[0].url=https://myartifactory.example.com/artifactory/ \
+--moderne.dx.artifactory[0].astQueryFilters[0]='"name":{"$match":"*-ast.jar"}' \
+--moderne.dx.artifactory[0].astQueryFilters[1]='"repo":{"$eq":"example-maven"}' \
+--moderne.dx.maven[0].url=https://myartifactory.example.com/artifactory/libs-releases-local \
+--moderne.dx.maven[0].localRepository=~/.moderne-maven \
+--moderne.dx.organization.reposCsv=/Users/MY_USER/Documents/repos.csv \
+--moderne.dx.organization.defaultCommitOptions=PullRequest,Branch,Direct \
+```
+</TabItem>
+</Tabs>
+
+### Step 6: (Optionally) Configure an organizations service
+
+You should create a dedicated organizations service if you want to:
+
+* Limit access to the organizations you've [previously defined](./configure-dx-organizations.md) so that some users only have access to some repositories OR
+* Customize commit messages by repository (e.g., adding a JIRA ticket to your commit messages based on the repository)
+
+If you desire this functionality, please follow the instructions in our [creating an organizations service guide](./dx-org-service.md).
+
+Below is an example of what the Moderne DX service run command might look like at the end of this step if you configured the service to use only configured recipe sources.
```

Comment:
This line feels out of place, was this meant to go after step 7?

Reply:
Ah just a bad copy/paste - thanks will fix.